controllers/kustomization_decryptor.go

```diff
@@ -92,6 +92,9 @@ var (
 		formats.Json:   []byte("\"mac\": \"ENC["),
 		formats.Yaml:   []byte("mac: ENC["),
 	}
+	// unsupportedFormat is used to signal no sopsFormatToMarkerBytes format was
+	// detected by detectFormatFromMarkerBytes.
+	unsupportedFormat = formats.Format(-1)
```

Comment:
Could this be a constant instead of a variable?

Reply:
Yes, `maxEncryptedFileSize` as well :-)